src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/SymbolMatcherTests.cs

```diff
@@ -990,5 +990,137 @@ class C : I<int, bool>
 
             Assert.Equal("I<System.Int32,System.Boolean>.Item", ((PropertySymbol)mappedProperty).MetadataName);
         }
+
+        [Fact]
+        public void Method_ParameterNullableChange()
+        {
+            var source0 = @"
+class C
+{
+    void M(string s) { }
+}";
+            var source1 = @"
+class C
+{
+    void M(string? s) { }
```

Comment:
>(string? s) [](start = 10, length = 11)

let's also make this method return `ref string?` and take another parameter with tuple type: `(string? a, dynamic? b) tuple`